docs/istio-jwt.md

```diff
@@ -0,0 +1,52 @@
+# API-Gateway JWT Handler Feature Toggle
```

Comment:
Since we have docs for Istio and Ory JWT handler inside it might be better to rename the file to `jwt-handler.md`?

Reply:
Hmm, the only thing with ory here is how to use feature toggle to enable it, but the APIRule usage is related to istio only.